<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>option</class>
 <widget class="QDialog" name="option">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>320</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>520</width>
    <height>320</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Option</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Icon.qrc">
    <normaloff>:/Icons/Option.png</normaloff>:/Icons/Option.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QListWidget" name="listWidgetOption">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>148</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>148</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>General</string>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="icon">
       <iconset resource="Icon.qrc">
        <normaloff>:/Icons/Option.png</normaloff>:/Icons/Option.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Language</string>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="icon">
       <iconset resource="Icon.qrc">
        <normaloff>:/Icons/language.png</normaloff>:/Icons/language.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Slideshow</string>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="icon">
       <iconset resource="Icon.qrc">
        <normaloff>:/Icons/Slideshow Start.png</normaloff>:/Icons/Slideshow Start.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Screenshot</string>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="icon">
       <iconset resource="Icon.qrc">
        <normaloff>:/Icons/Screenshot.png</normaloff>:/Icons/Screenshot.png</iconset>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="OptionGroupBox">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="title">
      <string notr="true">GroupBox</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QCheckBox" name="LoadPhotosFolder">
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Load Other Photos in Folder</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="KeepAspectRatioCheckBox">
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Keep Aspect Ratio</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QCheckBox" name="slsFullscreen">
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Go to Full Screen Mode</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="LanguageLabel">
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Language :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="LanguageComboBox">
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <item>
           <property name="text">
            <string>Automatic (System Language)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string notr="true">English</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string notr="true">پارسی</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string notr="true">Español</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string notr="true">中國傳統</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="SppedLabel">
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Speed :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="SlideshowSpeed">
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="value">
           <number>2</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="SecLabel">
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Sec</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="7" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>49</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="5" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="ScreenshotDelay">
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Screenshot delay :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="ScreenshotDelaySpinBox">
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>60</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="SecLabel_2">
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Sec</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="6" column="0">
       <widget class="QCheckBox" name="ScreenshotAtuoMinimizeCheckBox">
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Auto Minimize to tray</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="OptionButtonBox"/>
   </item>
  </layout>
  <zorder>OptionButtonBox</zorder>
  <zorder>listWidgetOption</zorder>
  <zorder>OptionGroupBox</zorder>
 </widget>
 <tabstops>
  <tabstop>listWidgetOption</tabstop>
  <tabstop>KeepAspectRatioCheckBox</tabstop>
  <tabstop>LoadPhotosFolder</tabstop>
  <tabstop>LanguageComboBox</tabstop>
  <tabstop>SlideshowSpeed</tabstop>
  <tabstop>slsFullscreen</tabstop>
  <tabstop>ScreenshotDelaySpinBox</tabstop>
  <tabstop>ScreenshotAtuoMinimizeCheckBox</tabstop>
 </tabstops>
 <resources>
  <include location="Icon.qrc"/>
 </resources>
 <connections/>
</ui>
